docs/guides/migrate-1.md

```diff
@@ -69,13 +96,32 @@ The following packages should be updated to reflect the following semver ranges:
 }
 ```
 
-Then run `npm install` to have these packages be upgraded.
+### Upgrade other dependencies
 
-Once you've upgraded CanJS and these ancillary packages your app should generally work. SSR is still disabled, but the front-end should be okay.
+You may be using other packages that are specifically made to work with CanJS 2.x, like `bit-tabs@0`.
+
+At this point, you should look for those other dependencies in your `package.json` and upgrade them, like so:
+
+```shell
+npm install bit-tabs@1 --save
+```
+
+Once you’ve upgraded CanJS and these ancillary packages, your app should generally work. SSR is still disabled, but the front-end should be okay.
```

Comment:
I'm not sure I follow.. if you are still on Steal 0.16 at this point then you would have already specified `"npmAlgorithm": "flat"` most likely, right?  I think for the sake of the guide we should assume that people are using the same version of Node as before.